sanic_jwt/__init__.py

```diff
@@ -15,11 +16,14 @@ def initialize(
     app.config.from_object(settings)
 
     if class_views is not None:
-        # TODO:
-        # - Run some verifications that class_views is formatted
-        #   ('<SOME ROUTE>', ClassInheritedFromHTTPMethodView)
         for route, view in class_views:
-            sanic_jwt_auth_bp.add_route(view.as_view(), route)
+            try:
+                if issubclass(view, HTTPMethodView) and isinstance(route, str):
+                    sanic_jwt_auth_bp.add_route(view.as_view(), route)
+                else:
+                    raise Exception("class_views should follow this format ('<SOME ROUTE>', ClassInheritedFromHTTPMethodView)")
```

Comment:
Instead of a generic exception, this could be a subclass of `SanicException` similar to the exceptsion in `sanic_jwt.exceptions`

Reply:
Thank you for feedback I will come back with a patch